### DISCUSSION TITLE
Pause Project Usage Easily

### DISCUSSION THREAD

**Discussion by SalladShooter** [Litlyx/litlyx#7] [Ideas]
I would love a way to pause the analytics/usage without commenting out the <script> tag, especially while working on projects as it is eating up my usage quickly. In one day of testing I went through 450 out of the 3,000 usage visits (on phyal).

A way to pause this would allow people to work on things (even after deployment on the user’s end) without going through all of their usage, and require them to pay when it wasn’t needed.

**Comment by AntonioVerdiglione1996** [comment]
We are taking this idea in consideration. Thank you so much for the feedback!
